### PR TITLE
Clean up ROT.Map.Rogue

### PR DESCRIPTION
**Pull request by ZLester**
Nothing specifically addressed in #62, though it looks like there were additional unused variables that were initialized but never used.

Summary:
Removed unused variables.
Fixed spelling error in "_drawCorridore" method.
Made code style more internally consistent.

### CODE REVIEW

File: src/map/rogue.js [hunk 1]
```diff
@@ -6,297 +6,283 @@
...
-ROT.Map.Rogue.prototype._connectUnconnectedRooms = function() {
-	//While there are unconnected rooms, try to connect them to a random connected neighbor 
+ROT.Map.Rogue.prototype._connectUnconnectedRooms = function () {
+	//While there are unconnected rooms, try to connect them to a random connected neighbor
 	//(if a room has no connected neighbors yet, just keep cycling, you'll fill out to it eventually).
 	var cw = this._options.cellWidth;
 	var ch = this._options.cellHeight;
-	
-	var randomConnectedCell;
+
```
Comment (ZLester) [inline src/map/rogue.js]: randomConnectedCell was initialized but never used.

File: src/map/rogue.js [hunk 2]
```diff
@@ -6,297 +6,283 @@
...
 	var roomh;
 	var roomWidth = this._options["roomWidth"];
 	var roomHeight = this._options["roomHeight"];
 	var sx;
 	var sy;
-	var tx;
-	var ty;
 	var otherRoom;
-	
+
```
Comment (ZLester) [inline src/map/rogue.js]: `tx` and `ty` were initialized but never used.

File: src/map/rogue.js [hunk 3]
```diff
@@ -318,41 +304,41 @@ ROT.Map.Rogue.prototype._getWallPosition = function(aRoom, aDirection) {
...
 	return [rx, ry];
-}
+};
 
 /***
 * @param startPosition a 2 element array
 * @param endPosition a 2 element array
 */
-ROT.Map.Rogue.prototype._drawCorridore = function (startPosition, endPosition) {
+ROT.Map.Rogue.prototype._drawCorridor = function (startPosition, endPosition) {
```
Comment (ZLester) [inline src/map/rogue.js]: `corridore` !== `corridor`

File: src/map/rogue.js [hunk 4]
```diff
@@ -422,9 +408,9 @@ ROT.Map.Rogue.prototype._createCorridors = function () {
 					wall = 1;
 					otherWall = 3;
 				}
-				
-				this._drawCorridore(this._getWallPosition(room, wall), this._getWallPosition(otherRoom, otherWall));
+
+				this._drawCorridor(this._getWallPosition(room, wall), this._getWallPosition(otherRoom, otherWall));
```
Comment (ZLester) [inline src/map/rogue.js]: This is the only reference to the spelling-corrected method name.

**Comment by ondras** [comment]
Thanks @ZLester for cleaning this up!

**Pull request merged**